data-raw/covid_pmfs.R

```diff
@@ -7,20 +7,20 @@ params <- wwinference::get_params(
 
 # Put it all together
 generation_interval <- withr::with_seed(42, {
-  wwinference::simulate_double_censored_pmf(
+  wwinference:::simulate_double_censored_pmf(
```

Comment:
@dylanhmorris Not sure what happened here but I think an extra `:` was added unintentionally?

Reply:
No this is necessary, since functions like `simulate_double_censored_pmf` not exported.

https://stackoverflow.com/questions/41582136/r-what-do-you-call-the-and-operators-and-how-do-they-differ